file_picker/sites.py

```diff
@@ -43,8 +44,7 @@ def register(self, model_or_iterable, class_=None, name=None, **options):
             self._registry.append({'name': name, 'picker': picker_class(name, model)})
 
     def get_urls(self):
-        from django.conf.urls import include,patterns,url
-        urlpatterns = patterns('', url(r'^$', self.primary, name='index'),)
+        urlpatterns = [url(r'^$', self.primary, name='index'), ]
```

Comment:
maybe put the initial declaration of urlpatterns on multiple lines like we usually do in urls.py, just for instant recognition?